components/formatter/ExternalLink.vue

```diff
@@ -0,0 +1,7 @@
+<script>
+export default { props: { value: { type: String, required: true } } };
+</script>
+
+<template>
+  <a :href="value">{{ value }}</a>
```

Comment:
Links to anything outside the dashboard should always have `rel="nofollow noopener noreferrer"`.  Probably `target="_blank"` by default too.